<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/db.ico</normaloff>:/icons/db.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
   font: 9pt &quot;Montserrat&quot;;
   color: rgb(255, 255, 255);
}

QFrame{
  background-color: rgb(31, 38, 79);
}

QStatusBar{
  background-color: rgb(31, 38, 79);
}

#actionsGrpBox &gt; QPushButton:hover:!pressed
{
  background-color: rgb(11, 103, 198);
}

#actionsGrpBox &gt; QPushButton:hover:pressed
{
  background-color: rgb(10, 89, 170);
}

#actionsGrpBox &gt; QPushButton{
  background-color: rgb(31, 94, 255);
  color: rgb(255, 255, 255);
  border: unset;
  border-radius: 4px;
  font-size:12px;
}

#actionsGrpBox &gt; QPushButton:disabled{
  background-color: rgb(185, 186, 189);
}

QToolTip {
  color: black;
}

QLabel {
  color: rgb(255, 255, 255);
}

QGroupBox {
  color: rgb(255, 255, 255);
}

QComboBox {
  color: rgb(255, 255, 255);
  border: 1px solid  rgb(255, 255, 255);
  background-color: rgb(31, 38, 79);
  padding: 2px;
}

QListView {
 border: unset;
}

QListView::item:selected:!active {
  color: rgb(255, 255, 255);
}

QListView::item:hover {
  background-color: rgb(10, 89, 170);
}

QListView::item:selected {
  background-color: rgb(11, 103, 198);
}

QListView{
  outline: none;
}

QScrollBar:vertical {              
    border: unset;
    background-color: rgb(31, 38, 79);
    width:10px;    
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    border-radius: 4px;
    background-color: rgb(11, 95, 181);
    min-height: 0px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal,
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    height: 0px;
    background: none;
}

QScrollBar::add-line:vertical {
    height: 0px;
}

QScrollBar::sub-line:vertical {
    height: 0px;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="actionsGrpBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="title">
          <string>Actions</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="textureBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Texture Editor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modelBtn">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Model Parts Editor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="materialBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Material Editor</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
